BeardedSpice/TuneInStrategy.m

```diff
@@ -0,0 +1,53 @@
+//
+//  TuneInStrategy.m
+//  BeardedSpice
+//
+//  Created by Cristian Yáñez on 14-01-15.
+//  Copyright (c) 2015 Tyler Rhodes / Jose Falcon. All rights reserved.
+//
+
+#import "TuneInStrategy.h"
+
+@implementation TuneInStrategy
+-(id) init
+{
+    self = [super init];
+    if (self) {
+        predicate = [NSPredicate predicateWithFormat:@"SELF LIKE[c] '*tunein.com*'"];
+    }
+    return self;
+}
+-(BOOL) accepts:(id <Tab>)tab
+{
+    return [predicate evaluateWithObject:[tab URL]];
+}
+-(NSString *) toggle
+{
+    return @"document.querySelector('.playbutton-cont').click();";
```

Comment:
So this code does toggle?

And np. If you couldn't tell by the code, we aren't either ;)